packages/shorebird_cli/lib/src/shorebird_artifact_mixin.dart

```diff
@@ -56,4 +56,23 @@ mixin ShorebirdArtifactMixin on ShorebirdCommand {
     await unzipFn(zipPath, extractedAarDir);
     return extractedAarDir;
   }
+
+  File newestAppDill() {
```

Comment:
I recommend adding a note explaining how much of a hack this is.  We don't actually know if this is even finding the correct dill for example. :)

Reply:
Would another option be to wipe out `.dart_tool/flutter_build` before generating the `app.dill` to ensure there is only a single one?

Reply:
We could also parse it out of the logs that flutter build makes. 🤷  I think the current "newest" hack is fine for now (we might end up just using the output from the ipa eventually, that just requires writing a mach-o parser), just wanted it marked as a hack.

Reply:
Added a comment :)